### ISSUE TITLE
LightGBM explain_weights() with split/weight does not work

### ISSUE THREAD

**Issue by paramdutta** [TeamHG-Memex/eli5#215]
Hi,
LightGBM explain_weights() with importance_type 'split' (/weight) returns empty feature_importances. 'gain' is working fine. 

Let me know if I am missing anything.

Thanks,
Param

**Comment by lopuhin** [comment]
hi @paramdutta , I just tested locally with the latest lightgmb master, and it seems to work for me.

Could you please check how feature importances directly returned from lightgbm work for you? E.g. ``gbm.feature_importance("gain")`` and ``gbm.feature_importance("split")``?

**Comment by kmike** [comment]
Tests from https://github.com/TeamHG-Memex/eli5/pull/222 fail in Python 2.x.
